pkgs/development/compilers/elm/packages/elm-instrument/default.nix

```diff
@@ -14,6 +14,7 @@ mkDerivation {
     rev = "63e15bb5ec5f812e248e61b6944189fa4a0aee4e";
     fetchSubmodules = true;
   };
+  patches = [ ./update-optparse-applicative.patch ];
```

Comment:
why not to use fetchpatch?

Reply:
Didn't know about it, it would allow to query the commit of the PR in the form of a patch I guess? I'll update the PR :)

Reply:
it's preferable because it makes it much easier to track if the patch is still necessary in the future.